src/main/java/com/onarandombox/MultiverseCore/destination/BedDestination.java

```diff
@@ -46,11 +47,11 @@ public boolean isThisType(JavaPlugin plugin, String destination) {
         boolean validFormat = split.length >= 1 && split.length <= 2 && split[0].equals(this.getIdentifier());
 
         OfflinePlayer p = Bukkit.getOfflinePlayer(split[1]);
-        boolean validPlayer = (p != null);
+        boolean validPlayer = p.getName() != null && !(p.getName().equals(OLD_BED_STRING) || p.getName().equals(CALLER_BED_STRING));
 
         if (validFormat && validPlayer) this.playername = p.getName();
 
-        this.isValid = destination.equals(OLD_BED_STRING) || (validFormat && validPlayer);
+        this.isValid = destination.equals("b:" + CALLER_BED_STRING) || destination.equals("b:" + OLD_BED_STRING) || (validFormat && validPlayer);
```

Comment:
I think we can just decide on one identifier to used as for the caller bed string. I think `playerbed` is ok already, since I googled on mcuuid and there is no such account name.

Reply:
Fair enough. I had just changed it to ensure that there would never be any conflicts, but its probably pretty unlikely that anyone would register and use the name `playerbed`.

Reply:
I kinda disagree. I think we should change it so that there is no possibility for conflict. @dumptruckman any opinion on this? I guess it doesn't really matter... the username `playerbed` is kinda useless.

Reply:
Actually, I think that would be a change for MV 5.